models/datasets/msf/ds__lab_follow_ups.yml

```
@@ -0,0 +1,51 @@
+version: 2
+
+models:
+  - name: ds__lab_follow_ups
+    description: MSF's lab follow-up schedule dataset for patients with non-communicable diseases (NCDs) conditions.
+    tags:
+      - clinical
```

Looks like there's been a change to the schema.
```suggestion
    config:
      tags:
        - clinical
```